### ISSUE TITLE
extra ')' in location

### ISSUE THREAD

**Issue by igorrosenberg** [chadn/mapfilter#11]
Load this:
http://chadnorwood.com/gcm/?ed=2014-01-01&gc=qanqkqs9d6iif7m8vmm2l6bar4%40group.calendar.google.com

In the drawer, in the where column, the location always ends with a ')'

**Comment by igorrosenberg** [comment]
There was just a typo. The fix is this (deletion of a character): 

```
bash> git diff examples/gcm/js/cnMapFilterUI.js
diff --git a/examples/gcm/js/cnMapFilterUI.js b/examples/gcm/js/cnMapFilterUI.js
index c9f16a8..e13c7c1 100644
--- a/examples/gcm/js/cnMapFilterUI.js
+++ b/examples/gcm/js/cnMapFilterUI.js
@@ -859,7 +859,7 @@ $(document).ready(function() {

                                rowHTML += onlyValidCoords 
                                        ? '<td><a class="actionable" href="javascript:void(0)" title="Original Address: ' 
-                                               + cnMFUI.htmlEncode(cnMFUI.maxStr(kk.addrOrig, 100, 0, '', 0)) + '">'+ kk.addrFromG
+                                               + cnMFUI.htmlEncode(cnMFUI.maxStr(kk.addrOrig, 100, 0, '', 0)) + '">'+ kk.addrFromG
                                        : '<td>'+(kk.addrOrig.match(/\w/) ? kk.addrOrig : '[empty]' );
                                rowHTML += '<br><a href="' + kk.url + '" title="Click to edit this event (if you have permission) i
                                        +' target="_blank">Edit Event Address</a><br>Error: ' + kk.error + '</td>';
```

**Comment by chadn** [comment]
Fixed in commit f7992a4 - Note that the comment above is cut off.  Instead of pasting a diff, read up on github's pull requests (in this case a pull request might be overkill, but good to do in general)

**Comment by igorrosenberg** [comment]
I'm fighting with a firewall at work, and am having difficulties to push to
github... So I was just storing the info, and had thought of pushing it
tonight from home... You beat me to it... Sorry about the hassle
